state-chain/pallets/cf-ingress-egress/src/boost_pool.rs

```diff
@@ -146,6 +146,15 @@ pub struct BoostPool<AccountId, C: Chain> {
 	pending_withdrawals: BTreeMap<AccountId, BTreeSet<PrewitnessedDepositId>>,
 }
 
+#[derive(DefaultNoBound, DebugNoBound, PartialEqNoBound)]
+pub struct DepositFinalisationOutcomeForPool<AccountId, C: Chain>
+where
+	AccountId: PartialEq + core::fmt::Debug,
+{
+	pub unlocked_funds: Vec<(AccountId, C::ChainAmount)>,
+	pub amount_credited: C::ChainAmount,
```

Comment:
Initially I assumed this included crediting the network fee to the protocol, before reading the rest of the code, since it didn't seem specific to boosters
```suggestion
	pub amount_credited_to_boosters: C::ChainAmount,
```